client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -52,8 +52,8 @@
             });
     }
 
-    CardsService.$inject = ['api', 'search', 'session'];
-    function CardsService(api, search, session) {
+    CardsService.$inject = ['api', 'search', 'session', 'desks'];
```

Comment:
It would be useful to add a docstring, summarizing in a few words what the `CardService` does.